app/src/main/java/com/redcoracle/episodes/AddShowSearchFragment.java

```diff
@@ -133,7 +133,7 @@ public void onLoaderReset(Loader<List<Show>> loader) {
 			List<Show> results = tvdbClient.searchShows(query, language);
 
 			// If there are no results, try searching all languages or substituting &
-			if(results.size() == 0 && query.contains(" and ")) {
+			if(results == null || (results.size() == 0 && query.contains(" and "))) {
 				results = tvdbClient.searchShows(query.replace(" and ", " & "), "all");
 			}
 			if(results.size() == 0) {
```

Comment:
I think line 139 could also raise an exception if it's still null from line 137. Maybe add an additional null check there while you're at it?